mockserver.js

```diff
@@ -247,6 +247,10 @@ function getDirectoriesRecursive(srcpath) {
  * GET--query=string&hello=hella.mock
  */
 function getBodyOrQueryString(body, query) {
+  if (body && query) {
+    return '_'+ body + '--' + query;
+  }
+
```

Comment:
This is the behavior described in the docs, but I don't think it was actually implemented. I'd appreciate another set of eyes on this.

If it's in doubt, i can remove it since it's not directly tied to the core purpose of the pr.